File: PRAK302-2510817320004-RahmaSyaritaJaya.py
nilai = int(input())
nilai_akhir = nilai // 10

if nilai_akhir >= 8:
    grade = 'A'
elif nilai_akhir == 7:
    grade = 'B'
elif nilai_akhir == 6:
    grade = 'C' 
elif nilai_akhir == 5:
    grade = 'D'
else:
    grade = 'E'

print(f"{grade}")

File: PRAK302-2510817320004-RahmaSyaritaJaya.c
#include <stdio.h>

int main (){
int nilai;
char grade;

scanf("%d", &nilai);

int nilaiAkhir = nilai / 10;
switch(nilaiAkhir){
case 10:
case 9:
case 8:
grade = 'A';
break;

case 7:
grade = 'B';
break;

case 6:
grade = 'C';
break;

case 5:
grade = 'D';
break;

default:
grade = 'E';
break;
}

printf("%c", grade);
return 0;
}